go/vt/vtctl/workflow/switcher_dry_run.go

```diff
@@ -380,7 +380,7 @@ func (dr *switcherDryRun) resetSequences(ctx context.Context) error {
 }
 
 func (dr *switcherDryRun) initializeTargetSequences(ctx context.Context, sequencesByBackingTable map[string]*sequenceMetadata) error {
-	sortedBackingTableNames := maps2.Keys(sequencesByBackingTable)
+	sortedBackingTableNames := maps.Keys(sequencesByBackingTable)
```

Comment:
Why was this change required?

Reply:
I think this is because `main` now has the `maps2` cleanup merged so it shows this diff here, but there's really no change. Merging in `main` or rebasing this change would resolve that I think.

Reply:
Rebasing with `main` has fixed the same!